deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -1 +1,73 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+@import url (http://localhost:63342/frontend-html-css-flexbox/deel-1-flexbox-positioneren/index.html?_ijt=dh5qmdfdq9ktueu66gq3obcar4&_ij_reload=RELOAD_ON_SAVE);
```

Comment:
Ik weet niet waarom hier een @import is gedefinieerd. Volgens mij is dit niet nodig in dit geval. Hoor anders graag waarom dit nodig is.